src/main/resources/db/migration/V7__add_address_contact_defendant_alias_tables.sql

```diff
@@ -0,0 +1,134 @@
+BEGIN;
+-------- address -------------------------------------------------------
+DROP TABLE IF EXISTS address;
+
+CREATE TABLE IF NOT EXISTS address
+(
+    id                                      SERIAL      PRIMARY KEY,
+    address_line_one                        TEXT        DEFAULT NULL,
+    address_line_two                        TEXT        DEFAULT NULL,
+    address_line_three                      TEXT        DEFAULT NULL,
+    address_line_four                       TEXT        DEFAULT NULL,
+    address_line_five                       TEXT        DEFAULT NULL,
+    postcode                                TEXT        DEFAULT NULL,
+    created_by                              TEXT        NOT NULL,
+    created_date                            TIMESTAMP   NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    last_updated_by                         TEXT        NOT NULL,
+    last_updated_date                       TIMESTAMP   NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    version                                 int4        NOT NULL DEFAULT 0
+
+);
+
+DROP TABLE IF EXISTS address_aud;
```

Comment:
I think we should drop this auditing and use hmpps_audit if we do have audit requirements (which at the moment for me we do not) - Phil kept complaining about it and it seems to cause us problems